controllers/clusters/postgresql_controller.go

```diff
@@ -1732,9 +1420,9 @@ func (r *PostgreSQLReconciler) SetupWithManager(mgr ctrl.Manager) error {
 					return false
 				}
 
-				oldObj := event.ObjectOld.(*v1beta1.PostgreSQL)
+				//oldObj := event.ObjectOld.(*v1beta1.PostgreSQL)
```

Comment:
Also this one should be deleted too. Please fix similar cases id there are

Reply:
fixed